tests/stdlib/tparsesql.nim

```diff
@@ -0,0 +1,521 @@
+import unittest
```

Comment:
No unit tests in testament. Use a ``discard`` "spec" section.

Reply:
I have switched to the discard spec format.